Gruntfile.js

```diff
@@ -46,8 +46,18 @@ module.exports = function(grunt) {
 				cmd(tag, suite, section) {
 					if (suite === 'integration') {
 						var slowTag = '';
-						if (tag !== 'slow') {
+						if (tag === 'untagged') {
 							slowTag = '--grep @slow --invert';
+						} else if (tag === 'extensive') {
+							slowTag = '--grep @unstable --invert';
+						} else if (tag === 'slow') {
+							slowTag = '--grep @slow';
+						} else if (tag === 'unstable') {
+							slowTag = '--grep @unstable';
+						} else {
+							grunt.fail.fatal(
+								'Please specify a test suite to run.\n\nExample: `grunt mocha:<tag>:<suite>:[section]` or `npm test -- mocha:<tag>:<suite>:[section]`\n\n- Where tag can be one of slow | unstable | untagged | extensive (required)\n- Where suite can be one of unit | functional | integration (required)\n- Where section can be one of get | post | ws | system (optional)'
```

Comment:
I think it is better to specify why we throw the error. I suggest to add at the beginning of the notice `The specified tag is not supported`.

In addition, I don't see anything related to Jenkins. Please @Nazgolze or @fchavant, be aware and provide some input if necessary.